internal/metrics/metrics.go

```diff
@@ -187,6 +227,172 @@ func GetScalingSignal(pdb *policyv1.PodDisruptionBudget) string {
 	return PDBBlockedSignal
 }
 
+// GetScalingOpportunityLabels gets age pattern and effectiveness labels for scaling opportunities
+func GetScalingOpportunityLabels(ctx context.Context, c client.Client, pdb *policyv1.PodDisruptionBudget) (string, string) {
+	pattern, err := AnalyzePodAgeFailurePattern(ctx, c, pdb)
+	if err != nil {
+		return "unknown", "unknown"
+	}
+	
+	isHelpful := PredictScalingEffectiveness(pattern)
+	helpfulLabel := UnlikelyHelpfulStr
+	if isHelpful {
+		helpfulLabel = LikelyHelpfulStr
+	}
+	
+	return pattern, helpfulLabel
+}
+
+// AnalyzePodAgeFailurePattern analyzes the failure pattern of pods based on their age
+func AnalyzePodAgeFailurePattern(ctx context.Context, c client.Client, pdb *policyv1.PodDisruptionBudget) (string, error) {
+	// Convert PDB label selector to Kubernetes selector
+	selector, err := metav1.LabelSelectorAsSelector(pdb.Spec.Selector)
+	if err != nil {
+		return AllHealthyPattern, err
+	}
+
+	// List all pods matching the PDB selector
+	var podList corev1.PodList
+	err = c.List(ctx, &podList, &client.ListOptions{
+		Namespace:     pdb.Namespace,
+		LabelSelector: selector,
+	})
+	if err != nil {
+		return AllHealthyPattern, err
+	}
+
+	if len(podList.Items) == 0 {
+		return AllHealthyPattern, nil
+	}
+
+	// Sort pods by creation time 
+	sort.Slice(podList.Items, func(i, j int) bool {
+		return podList.Items[i].CreationTimestamp.Before(&podList.Items[j].CreationTimestamp)
+	})
+
+	// Categorize pods as unhealthy
+	var unhealthyPods []corev1.Pod
+	for _, pod := range podList.Items {
+		if !isPodHealthy(&pod) {
+			unhealthyPods = append(unhealthyPods, pod)
+		}
+	}
+
+	// If all pods are healthy
+	if len(unhealthyPods) == 0 {
+		return AllHealthyPattern, nil
+	}
+
+	// Analyze failure pattern 
+	totalPods := len(podList.Items)
+	totalUnhealthyPods := len(unhealthyPods)
+	
+	// If less than 2 pods, can't determine pattern clearly
```

Comment:
handle this early with if len(podList.Items) == 0 

This means thay have o have a minavailable = 1 or they woudn't be blocked?